src/NServiceBus.Metrics.PerformanceCounters/PerformanceCountersFeature.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Features;
+
+class PerformanceCountersFeature : Feature
+{
+    public PerformanceCountersFeature()
+    {
+        Defaults(s =>
+        {
+            options = s.EnableMetrics();
+            s.SetDefault(UpdateIntervalKey, TimeSpan.FromSeconds(2));
+        });
+    }
+
+    protected override void Setup(FeatureConfigurationContext context)
+    {
+        var logicalAddress = context.Settings.LogicalAddress();
+
+        var legacyInstanceNameMap = new Dictionary<string, CounterInstanceName?>
```

Comment:
Ok gotcha. That would mean all things that use the meters and timers currently defined in Metrics pkg would depend on that typo and abbrevation as well. I'd rather have that mapping logic. Thoughts?